templates/pbuilderrc.erb

```diff
@@ -99,8 +99,8 @@ if [ -n "${BUILDMIRROR}" ]; then
   OTHERMIRROR=$BUILDMIRROR
 
   # Add internal signing key
-  DEBOOTSTRAPOPTS=("${DEBOOTSTRAPOPTS[@]}" "--keyring=/usr/share/keyrings/pluto-build-keyring.gpg")
-  APTKEYRINGS=("${APTKEYRINGS[@]}" "/usr/share/keyrings/pluto-build-keyring.gpg")
```

Comment:
I have no idea why we still have references to the pluto keyring here. @haus @MosesMendoza is this a terrible change? Are we still using this key for anything?